repositories.yaml

```diff
@@ -1381,3 +1381,23 @@ applications:
     moz_pipeline_metadata_defaults:
       expiration_policy:
         delete_after_days: 760
+
+  - app_name: moso_mastodon_backend
```

Comment:
I'm assuming from the Bugzilla ticket that this is for collecting backend data using Glean core? Which is why I changed the app ID to `moso.mastodon.backend`. I'm guessing we'll also want to collect frontend data using glean.js at some point for which we'd add a new app with the ID `moso.mastodon.frontend`.

Does this sound correct, @braunreyes?

Reply:
Mastodon is a server-side html Rails app I just went with moso.mastodon since we are probably never going to instrument the front end aspects of the mastodon app.  That being said I am perfectly fine with being explicit here and can create a PR to update the mastodon code base as needed.  `moso.mastodon.backend` seems fine to me.

Reply:
I see, being consistent and adding a `_backend` is I think preferable here. We have done the same for FxA and some other applications as well.